app/scripts/services/unit.coffee

```diff
@@ -347,6 +347,17 @@ angular.module('swarmApp').factory 'Unit', (util, $log, Effect, ProducerPaths, U
     #util.assert max.greaterThanOrEqualTo(0), "invalid unit cost max", @name
     return max
 
+  _costMetPerSecond: ->
+    max = new Decimal Infinity
+    console.log max
+    for cost in @eachCost()
+      if cost.val.greaterThan(0)
+        console.log cost,cost.val.toNumber(),cost.unit.velocity().toNumber(),cost.unit.velocity().dividedBy cost.val
```

Comment:
Oops, I actually meant to remove all debug logging, I just missed this one. Will pull it.